gulpfile.js

```diff
@@ -134,6 +135,13 @@ gulp.task('tests', ['lint'], function () {
     });
 });
 
+gulp.task('karma', function () {
```

Comment:
I'm not so sure yet, want to play around a bit more and involve @Marketionist with the testing setup before doing so. Sounds alright?